.github/workflows/docker.yml

```diff
@@ -16,13 +16,15 @@ jobs:
       # Prevent a failure in one image from stopping the other builds
       fail-fast: false
       matrix:
+        platforms: ["linux/arm/v7", "linux/amd64", "linux/arm64"]
```

Comment:
This approach will start one job per architecture, and seems like it broke the matrix entirely. I expect `["linux/arm/v7,linux/amd64,linux/arm64"]` should work, while delegating the multi-arch to inside the build-push-action.